.github/workflows/proposal-review.yaml

```diff
@@ -211,9 +211,9 @@ jobs:
         env:
           OBJECT_ID: ${{ steps.fetch.outputs.proposalId }}
           PROPOSAL_STATE: ${{ steps.fetch.outputs.proposalState }}
-          ALGOLIA_APP_ID: ${{ secrets.NEXT_PUBLIC_ALGOLIA_APP_ID }}
-          ALGOLIA_ACCESS_KEY: ${{ secrets.ALGOLIA_SEARCH_ADMIN_KEY }}
-          ALGOLIA_INDEX_NAME: ${{ secrets.NEXT_PUBLIC_ALGOLIA_INDEX_NAME }}
+          ALGOLIA_APP_ID: ${{ secrets.ALGOLIA_APP_ID }}
+          ALGOLIA_ACCESS_KEY: ${{ secrets.ALGOLIA_WRITE_API_KEY }}
```

Comment:
we could also rename this env variable keys and update the usage in down in the command.

Reply:
fair point, alrighty 🙌